Firmware/temperature.cpp

```diff
@@ -1427,20 +1451,47 @@ enum { LCDALERT_NONE = 0, LCDALERT_HEATERMINTEMP, LCDALERT_BEDMINTEMP, LCDALERT_
 //! to prevent flicker and improve speed
 uint8_t last_alert_sent_to_lcd = LCDALERT_NONE;
 
+
+//! update the current temperature error message
+//! @param type short error abbreviation (PROGMEM)
+//! @param func optional lcd update function (lcd_setalertstatus when first setting the error)
+void temp_update_messagepgm(const char* PROGMEM type, void (*func)(const char*) = lcd_updatestatus)
+{
+    char msg[LCD_WIDTH];
+    strcpy_P(msg, PSTR("Err: "));
```

Comment:
interestingly, filling the msg this way results in 30B shorter code than in `char msg[] = "Err: ";`

Reply:
Tried a few different approaches, I was surprised too. It's a bit of a gimmick to pass the update function, but I didn't want to duplicate the str* function twice.

When I was looking to unify some code I think we could actually also remove the "Err: " prefix in a few other places by adding a proper lcd function to set error messages. But it had a bit too many changes for this PR. Something to think about for the future though.